packages/components/src/Menu/Menu.tsx

```diff
@@ -208,15 +214,30 @@ export interface ActionProps {
    * Callback when an action gets clicked
    */
   onClick?(event: React.MouseEvent<HTMLButtonElement>): void;
+
+  /**
+   * Focuses explicitly if set to true
+   */
+  shouldFocus?: boolean;
 }
 
-function Action({ label, icon, onClick }: ActionProps) {
+function Action({ label, icon, onClick, shouldFocus = false }: ActionProps) {
+  // eslint-disable-next-line no-null/no-null
+  const actionButtonRef = useRef<HTMLButtonElement | null>(null);
```

Comment:
Let's be consistent with how we get around the ESLint rule for `null` using type casting:
```suggestion
  const actionButtonRef = useRef() as RefObject<HTMLButtonElement>;
```

Reply:
This is how we do it right now in other components, so should be good to match. This might not be the best approach long term but we shouldn't tackle that here 😃

Reply:
Definitely! makes sense for the consistency. TS can sometimes really go against the wind and tell you NOT to use JS/React the way they are designed to be used 😄. Will update. Thanks 🙂!